content/md/what-is/what-is-a-category.md

```diff
@@ -31,11 +31,10 @@ A category or a category tree only needs a code to be created. You can name your
 
 ## From the user interface
 
-Go to `Settings`/`Categories`. To create a new category tree, click on `Create a new category` button.
+Go to `Settings`/`Categories`. To create a new category tree, click on `Create` button.
 ![How to create a new category](../img/Settings_CreateCategoryCTA.png)
```

Comment:
In this screenshot, there is no "create" button as explained in the paragraph just over it?